docs/DeveloperGuide.md

```diff
@@ -171,6 +174,41 @@ As such, the detailed descriptions for the Address Book subsystem above can be t
 
 This section describes some noteworthy details on how certain features are implemented.
 
+### Import and Export CSV Features
+
+This section describes some of the details as to how the import and export CSV features were implemented
+
+#### Import CSV
+
+The import CSV function is meant to append to the current address book with new data imported from any CSV file. The intention is to allow users to be able to import from a Microsoft Excel compatible format. Since there are multiple different templates for contacts in CSV files across various platforms, such as Microsoft Outlook and Google Contacts, the idea was to ensure this feature was as flexible as possible, allowing the user to specify which particular columns contained certain information
```

Comment:
I would suggest a change in phrasing here.
```suggestion
The import CSV function is meant to append to the current address book with new data imported from any CSV file. The intention is to allow users to be able to import from a Microsoft Excel compatible format. Since there are multiple different templates for contacts in CSV files across various platforms, such as Microsoft Outlook and Google Contacts, the feature is designed to be as flexible as possible, allowing the user to specify mappings for the information contained in the various columns.
```

Reply:
thanks for the suggestion! implemented in https://github.com/AY2122S2-CS2103-W17-1/tp/pull/203/commits/3193b02116aeddfa74a438d365ea5575d9acfb0b